controllers/common.go

```diff
@@ -342,12 +342,6 @@ func (e *Endpoint) Health(ctx *gin.Context) {
 	ctx.HTML(http.StatusOK, "index.html", gin.H{})
 }
 
-// @Description Error response description
-type ErrorResponse struct {
-	// Textual description of encountered error
-	Error string `json:"error" example:"message"`
-} // @name ErrorResponse
-
```

Comment:
The response models are defined in core, so it was never among the response models.

Reply:
?
ErrorResponse is literally a model we return if request errors. It is there in the docs.

And what core are you talking about?

Reply:
The models for other stuff such as SliceMetadata are defined in internal/core

Reply:
Ah, ok.
In swagger docs all of them are just "models".
Models come from request models in "api" and from response models that were both in "response" and in "core". Additionally there are some request-response defining models that are also defined in core.

We never gave any thought to "these are response models so they should belong here and there"

So we will have to think if we want to make a bigger restructure with "models" directory creation.

With current structure this class to me still belongs into response.go file rather than in middleware. Middleware in my opinion is something that can be dismissed/exchanged if wanted. I can't see any user-facing models belonging there.

Reply:
I don't think we need a models directory as this is typically used for models corresponding to core concepts in your "business logic", not requests and responses to the API endpoints.

We could consider having separate structs for these in the API code though to allow them to evolve independently of the structs in core. We could then for instance add annotations used for swagger docs, something that would be a bit strange to see in core.